src/auro-calendar-cell.js

```diff
@@ -25,6 +27,7 @@ export class AuroCalendarCell extends LitElement {
     this.isCurrentDate = false;
     this._locale = null;
     this.dateStr = null;
+    this.renderForDateSlot = false;
```

Comment:
We need to be in better practice of explaining our code using comments. Since this is a private attribute it's never explained elsewhere.

```suggestion
    this.renderForDateSlot = false; // When false, the numerical date will render vertically centered. When true, the date will render offcenter to the top and leave room below for the slot content.
```